docs/GM-Commands.md

```diff
@@ -373,150 +456,68 @@ Note: Some commands are working only by selecting a player or a creature. These
+|server set difftime|4|Syntax: .server set difftime $time. $time (in milliseconds) update time diff is written to the log file.|
+|server set loglevel|4|Syntax: .server set loglevel $facility $name $loglevel. $facility can take the values: appender (a) or logger (l). $loglevel can take the values: disabled (0), trace (1), debug (2), info (3), warn (4), error (5) or fatal (6)|
 |server set motd|3|Syntax: .server set motd $MOTD Set server Message of the day.|
+|server shutdown cancel|3|Syntax: .server shutdown cancel Cancel the restart/shutdown timer if any.|
+|server shutdown|3|Syntax: .server shutdown #delay [#exit_code] Shut the server down after #delay seconds. Use #exit_code to change the exit code, default = 0.|
 |server togglequerylog|4|Syntax: .server togglequerylog. Toggle SQL query log on/off.|
-|server|3|Syntax: .server $subcommand Type .server to see the list of possible subcommands or .help server $subcommand to see info on subcommands|
 |setskill|3|Syntax: .setskill #skill #level [#max] Set a skill of id #skill with a current skill value of #level and a maximum value of #max (or equal current maximum if not provide) for the selected character. If no character is selected, you learn the skill.|
 |showarea|3|Syntax: .showarea #areaid Reveal the area of #areaid to the selected character. If no character is selected, reveal this area to you.|
+|hidearea|3|Syntax: .hidearea #areaid Hide the area of #areaid to the selected character. If no character is selected, hide this area to you.|
+|skirmish|3|Syntax: .skirmish [arena] [XvX] [Nick1] [Nick2] ... [NickN]. [arena] can be "all" or comma-separated list of possible arenas (NA,BE,RL,DS,RV). [XvX] can be 1v1, 2v2, 3v3, 5v5. After [XvX] specify enough nicknames for that mode.|
 |spect leave|0|Syntax: .spect leave. Leave arena spectator.|
 |spect reset|0|Syntax: .spect reset. Reset arena spectator.|
 |spect spectate|0|Syntax: .spect spectate #playername. Start arena spectator, watch #playername.|
 |spect version|0|Syntax: .spect version. Check spectator addon version.|
 |spect watch|0|Syntax: .spect watch #playername. Watch #playername.|
-|skirmish|3|Syntax: .skirmish [arena] [XvX] [Nick1] [Nick2] ... [NickN]. [arena] can be "all" or comma-separated list of possible arenas (NA,BE,RL,DS,RV). [XvX] can be 1v1, 2v2, 3v3, 5v5. After [XvX] specify enough nicknames for that mode.|
-|unstuck|0|Syntax: .unstuck $playername [inn/graveyard/startzone] Teleports specified player to specified location. Default location is player's current hearth location.|
-|cheat god|2|Syntax: .cheat god [on/off] Enables or disables your character's ability to take damage.|
-|cheat casttime|2|Syntax: .cheat casttime [on/off] Enables or disables your character's spell cast times.|
+|summon|1|Syntax: .summon [$charactername] Teleport the given character to you. Character can be offline.|
+|tele|1|Syntax: .tele #location Teleport player to a given location.|
```

Comment:
```suggestion
|teleport|1|Syntax: .teleport #location Teleport player to a given location.|
```

I think we now have the full command name thanks to @Kitzunu (.tele works IG too though)